notary/server/README.md

```diff
@@ -109,7 +109,7 @@ To perform notarization, some parameters need to be configured by the prover and
 To streamline this process, a single HTTP endpoint (`/session`) is used by both TCP and WebSocket clients.
 
 #### Notarization
-After calling the configuration endpoint above, prover can proceed to start notarization. For TCP client, that means calling the `/notarize` endpoint using HTTP (`https`), while WebSocket client should call the same endpoint but using WebSocket (`wss`). Example implementations of these clients can be found in the [integration test](./tests/integration_test.rs).
+After calling the configuration endpoint above, prover can proceed to start notarization. For TCP client, that means calling the `/notarize` endpoint using HTTP (`https`), while WebSocket client should call the same endpoint but using WebSocket (`wss`). Example implementations of these clients can be found in the [integration test](../tests-integration/tests/notary.rs).
```

Comment:
I have a general comment that the "telegram style" which is acceptable when commenting the code where we omit the articles "a , the" is not appropriate imo for docs like this README. It seems like in this README there is a mixture of both style. I'll just fix this section to show what I mean (maybe someone go over our docs later and fix the rest)

```suggestion
After calling the configuration endpoint above, the prover can proceed to start the notarization. For a TCP client, that means calling the `/notarize` endpoint using HTTP (`https`), while a WebSocket client should call the same endpoint but using WebSocket (`wss`). Example implementations of these clients can be found in the [integration test](../tests-integration/tests/notary.rs).
```

Reply:
Good spot!